### PR TITLE
DOCS-1829: Make carousel shortcode and rearrange Build

### PR DESCRIPTION
**Pull request by JessamyT**
Note to reviewers: The only change to the big HTML chunk in Build is changing services h2 to h3.

### CODE REVIEW

File: docs/build/_index.md [hunk 1]
```diff
@@ -16,40 +16,46 @@ Components drive your hardware, and services provide high-level software functio
...
+<br><br>
 
-2. Then pull together the Viam resources you want to use by [configuring your smart machine](/build/configure/).
+{{< board-carousel >}}
 
-3. Finally, [program your smart machine](/build/program/) with an SDK in your preferred coding language.
+## Step 2: Configure
 
-{{% /alert %}}
+Pull together the Viam {{< glossary_tooltip term_id="resource" text="resources" >}} you want to use by [configuring your smart machine](/build/configure/).
```
Comment (npentrel) [inline docs/build/_index.md:30]:
```suggestion
Select the Viam {{< glossary_tooltip term_id="resource" text="resources" >}} you want to use by [configuring your smart machine](/build/configure/).
```

File: docs/build/_index.md [hunk 2]
```diff
@@ -16,40 +16,46 @@ Components drive your hardware, and services provide high-level software functio
...
     </div>
   </div>
 </div>
 
 If a component or service you want to use for your project is not natively supported, see the [modular resource registry](/registry/).
```
Comment (npentrel) [inline docs/build/_index.md:46]: Out of diff but noticed this soudns odd
```suggestion
If a component or service you want to use for your project is not natively supported, see whether it is supported as a {{< glossary_tooltip term_id="modular resource" text="modular-resource" >}} in the [registry](/registry/) or build your own modular resource.
```

File: docs/get-started/_index.md [hunk 3]
```diff
@@ -7,119 +7,7 @@ layout: "empty"
 empty_node: true
```
Comment (npentrel) [inline docs/get-started/_index.md:7]: This is actually not an empty node anymore and it should also not have the `layout: "empty"`
```suggestion
```

**Review by npentrel — APPROVED** [review]

**Comment by npentrel** [comment]
Great work - a few small comments

File: docs/get-started/_index.md (hunk 3 above)
Comment (JessamyT) [reply docs/get-started/_index.md:7]: Ah okay I commented out both of those lines temporarily to test it locally but figured you'd had these lines in here for a reason so left as-is! I'll change them

**Pull request merged**